src/AppRoutes.tsx

```diff
@@ -326,14 +325,7 @@ export const AppRoutes = (
           <Route path=":buildPushId" element={<BuildPushPage />} handle={breadcrumbData.buildPush.id} />
         </Route>
         <Route path="build-metrics" element={<BuildMetricsPage />} />
-        <Route
-          path="artifact-dependency-graph"
-          element={
-            <ExperimentalContent isRouteVariant>
-              <BuildArtifactDependencyGraphPage />
-            </ExperimentalContent>
-          }
-        />
+        <Route path="implicit-dependency-graph" element={<BuildImplicitDependencyGraphPage />} />
```

Comment:
Is there any specific reason that `<ExperimentalContent>` was removed?

Reply:
similarly to interconnection graph, only small hint on the page tab link was left to mark it as experimental